test/fuzz/lua/net_box_call_test.lua

```diff
@@ -0,0 +1,25 @@
+--[[
+"One byte of death"
+https://github.com/tarantool/tarantool/issues/6781
+]]
+
+local luzer = require("luzer")
+local net_box = require("net.box")
+
+local function TestOneInput(buf)
+    -- The test generates memtx snapshot, cleanup before the test.
+    os.execute("rm -f *.snap")
+    local socket_path = os.tmpname()
+    box.cfg({
+        listen = socket_path,
+        -- Suppress Tarantool diagnostic messages.
+        log_level = "warn",
+    })
```

Comment:
It's better to add a working directory for snapshots and logs. Also, it may be cleared much easier. Looks like we need to call `box.cfg{}` only once on the setup. See the previous comment.